templates/corporate/upgrade.html

```diff
@@ -153,17 +161,23 @@ <h1>$<span class="due-today-price"></span></h1>
                             {% if not manual_license_management and using_min_licenses_for_plan %}
                                 <i>Minimum purchase for this plan: {{ min_licenses_for_plan }} licenses</i>
                             {% endif %}
+                            {% endif %}
```

Comment:
I expect you'll want the indent the block inside this new outer if/else block.

Reply:
Updated the indentation. 

Actually, the indentation in our templates are kind of mixed-up. At some places we prefer indentation & at others we don't. Any technical constraint due to which we haven't added lint rule to check indentation in these files ?

Reply:
There actually is a lint rule -- called `templates` if you list the linters -- it just has a slightly more complex algorithm that allows blocks that don't contain additional control elements to not be intended, to avoid excessive/unhelpful nesting.